src/styles/bloks/product.scss

```diff
@@ -0,0 +1,92 @@
+.product {
+  box-sizing: border-box;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+
+  &__img {
+    margin-bottom: 40px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    margin-top: 0;
+
+    color: $mainColor;
+    font-size: 12px;
+    font-style: normal;
+    font-weight: 500;
+    line-height: 18px;
+  }
+
+  &__code {
+    margin-top: 0;
+    margin-bottom: 16px;
+
+    color: #616070;
+    font-size: 10px;
+    font-style: normal;
+    font-weight: 400;
+    line-height: 14px;
+  }
+
+  &__reviews {
+    margin-bottom: 24px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    justify-content: space-between;
+
+    &-text {
+      margin: 0;
+
+      color: #616070;
+      font-size: 12px;
+      font-style: normal;
+      font-weight: 400;
+      line-height: 18px;
+    }
+
+    &-value {
+      margin: 0;
+      font-size: 16px;
+      font-weight: 700;
+      line-height: 18px;
+    }
+  }
+
+  &__btn {
+    box-sizing: border-box;
```

Comment:
please give border-box for all items in the page using `*` selector and remove it from here and from `product`